### PR TITLE
[DC-1390] Add cdr_qa_report_dateshift.py script into run_cdr_qa_repor…

### PR DESCRIPTION
**Pull request by jqian2015**
Add cdr_qa_report_dateshift.py script into run_cdr_qa_report_dateshift.py, which is based on rdr_report_qc.py and changed some parameters.

### CODE REVIEW

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py [hunk 1]
```diff
@@ -0,0 +1,693 @@
...
+#     name: python3
+# ---
+
+# + papermill={"duration": 0.023643, "end_time": "2021-02-02T22:30:31.880820", "exception": false, "start_time": "2021-02-02T22:30:31.857177", "status": "completed"} tags=["parameters"]
+project_id = ""
+com_cdr = ""
+new_cdr = ""
+#dataset_id=""
+
+# + papermill={"duration": 0.046903, "end_time": "2021-02-02T22:30:31.927723", "exception": false, "start_time": "2021-02-02T22:30:31.880820", "status": "completed"} tags=["injected-parameters"]
```
Comment (jp3477) [inline data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:21]: You can get rid of this cell because it has project information.

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py [hunk 2]
```diff
@@ -0,0 +1,693 @@
...
+ON
+
+d.observation_period_id = i.observation_period_id
+
+limit 5
+
+'''
+pd.read_gbq(query, dialect='standard')
+
+# + [markdown] papermill={"duration": 0.023649, "end_time": "2021-02-02T22:30:39.115495", "exception": false, "start_time": "2021-02-02T22:30:39.091846", "status": "completed"} tags=[]
```
Comment (jp3477) [inline data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:126]: These papermill execution logs should also not be committed to the repo. I think they should be able to be cleared, but let me know if you can't.

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py (hunk 1 above)
Comment (jqian2015) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:22]: one quick question, if that cell is removed, how do we know what database or dataset should be used? from the queries i got from QA team, each sheet may use different datasets.

File: data_steward/analytics/cdr_ops/run_rdr_qc_report.py [hunk 3]
```diff
@@ -5,25 +5,25 @@
 import papermill as pm
```
Comment (jp3477) [inline data_steward/analytics/cdr_ops/run_rdr_qc_report.py:5]: You can remove this file, as this functionality will be handled by the report runner.

File: data_steward/analytics/cdr_ops/report_runner.py [hunk 4]
```diff
@@ -265,7 +265,7 @@ def __call__(self, value, **kwargs):
 
 
 if __name__ == '__main__':
-    pipeline_logging.configure(logging.INFO, add_console_handler=True)
+   # pipeline_logging.configure(logging.INFO, add_console_handler=True) #q2
```
Comment (jp3477) [inline data_steward/analytics/cdr_ops/report_runner.py:268]: I know this eliminated an error for you, but can you revert these changes to the report_runner? You can always comment it out for your local runs.

**Review by jp3477 — CHANGES_REQUESTED** [review]

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py (hunk 2 above)
Comment (jqian2015) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:126]: i am not sure about this part. basically, i use Jupiter notebook to edit and py file is saved in the fly. i could not see these execution logs in the notebook but do see this in the py file. not really sure what I can do.

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py (hunk 1 above)
Comment (jp3477) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:22]: You keep the first `parameters` cell, but not the second `injected parameters` cell that is output by papermill. That second cell has passed in parameter values that we do not want to place in the repo. Does that make sense?

File: data_steward/analytics/cdr_ops/run_rdr_qc_report.py (hunk 3 above)
Comment (jqian2015) [reply data_steward/analytics/cdr_ops/run_rdr_qc_report.py:5]: i don't see this line in my updated file.

Comment (jqian2015) [reply data_steward/analytics/cdr_ops/run_rdr_qc_report.py:5]: i see. we can forget about this file. i used runner_report.

Comment (jqian2015) [reply data_steward/analytics/cdr_ops/run_rdr_qc_report.py]: i removed this file.

File: data_steward/analytics/cdr_ops/report_runner.py (hunk 4 above)
Comment (jqian2015) [reply data_steward/analytics/cdr_ops/report_runner.py]: i see. ok. changed back.

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py (hunk 2 above)
Comment (jp3477) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:126]: Hmm, I think I see what's happening. The notebook runner updates the input notebook if no `--output_path` is passed. To avoid this , you could pass a `--output_path` with a different name than your input notebook.

Comment (jp3477) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:126]: Can you pull your clean, un-run version of your jupytext notebook from anywhere?

File: data_steward/analytics/cdr_ops/run_rdr_qc_report.py (hunk 3 above)
Comment (jp3477) [reply data_steward/analytics/cdr_ops/run_rdr_qc_report.py]: Thanks

File: data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py (hunk 2 above)
Comment (jqian2015) [reply data_steward/analytics/cdr_ops/cdr_qa_report_dateshift.py:117]: yes, done. now i understood what you were saying. thanks.

**Review by jp3477 — APPROVED** [review]
Approved!
